hackernews/program.py

```diff
@@ -0,0 +1,57 @@
+"""Monitor Hacker News for new articles on a specific topic, and post updates to a Slack channel."""
+
+import os
+import requests
+import time
+import urllib.parse
+
+from autokitteh.slack import slack_client
+
+
+slack = slack_client("slack_connection")
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+API_URL = "http://hn.algolia.com/api/v1/search_by_date?tags=story&page=0&query="
```

Comment:
Based on their names, `SLACK_CHANNEL` and `API_URL` are "const" variables, but `slack` is a "normal" global variable.

As such they should be ordered like this:

```
SLACK_CHANNEL = ...
API_URL = ...

slack = ...
```

Reply:
Also, I don't see any actual usage of `SLACK_CHANNEL` - in that can you can remove it from here and from the manifest